client/lib/route/index.ts

```diff
@@ -49,11 +42,12 @@ export const addQueryArgs = ( args, url ) => {
 	return urlModule.format( updatedUrlObject );
 };
 
-export const trailingslashit = path => path.replace( /(\/)?$/, '/' );
+export const trailingslashit = ( path: string ): string => path.replace( /(\/)?$/, '/' );
 
-export const untrailingslashit = path => ( path === '/' ? path : path.replace( /\/$/, '' ) );
+export const untrailingslashit = ( path: string ): string =>
+	path === '/' ? path : path.replace( /\/$/, '' );
 
-export const normalize = ( context, next ) => {
+export const normalize: PageJS.Callback = ( context, next ) => {
```

Comment:
Forgot to ask (since this is obviously working nicely), but how does TypeScript know about the `PageJS.Callback` type? We don't seem to be importing the `PageJS` symbol anywhere :thinking:

Reply:
To be honest, I'm not entirely sure 😅 I thought that was an error but it's apparently not 🤷‍♂ 

I'm not sure how that namespace becomes available.

That did lead me to spot #35004, however 😉